code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -334,7 +334,9 @@ This function restores all organs.
 
 	//Handle BRUTE and BURN damage
 	handle_suit_punctures(damagetype, damage)
-
+	//Handle species apply_damage procs
+	dna.species.spec_apply_damage(damage, damagetype, def_zone, blocked, sharp, used_weapon)

```

Comment:
In general I'd prefer to follow what TG do by directly overriding apply_damage and have that set their regen timer. It make converting code easier and ensure behavior are consistent.